api/kessel/relations/v1beta1/relation_tuples.proto

```diff
@@ -34,12 +34,26 @@ service KesselTupleService {
 			delete: "/v1beta1/tuples"
 		};
 	};
+	rpc ImportBulkTuples (stream
+			ImportBulkTuplesRequest) returns (ImportBulkTuplesResponse) {
+		option (google.api.http) = {
+			post: "/v1beta1/tuples/bulkimport"
+			body: "*"
+		};
+	};
+}
+
+message ImportBulkTuplesRequest {
+	repeated Relationship tuples = 1 [(buf.validate.field).required = true];
```

Comment:
SpiceDB uses a different validation rule here. I'm curious why this is different?

`[ (validate.rules).repeated .items.message.required = true ]`

Reply:
fixed